src/shared/utils/duration.ts

```diff
@@ -9,6 +9,9 @@ import {TIME_RANGE_FORMAT} from 'src/shared/constants/timeRanges'
 import {createDateTimeFormatter} from 'src/utils/datetime/formatters'
 
 export const removeSpacesAndNow = (input: string): string =>
```

Comment:
Yes, you're right. I'll just inline it.

Originally, there was already another function called `removeSpacesAndNow` but that function wasn't named correctly, because it was also removing the "-" or "+" sign. So i renamed that one and created this new one.

What I will do is remove this new one and inline it as you suggested. But I am keeping the renamed function to be clear what that other one is actually doing.